docs/plugins/language.md

```diff
@@ -0,0 +1,207 @@
+# Language
+
+The Language plugin helps to make your App multilanguage.
```

Comment:
The language plugin supports internationalisation features for your application?

Reply:
changed